### PR TITLE
feat(blockchain-api): account balances

### PR DESCRIPTION
**Pull request by geekbrother**
This PR adds specifications for the Account Balances request and response to the Blockchain-API service.

### CODE REVIEW

File: docs/specs/servers/blockchain/blockchain-server-api.md [hunk 1]
```diff
@@ -276,3 +276,37 @@ The POST request body should be in JSON format with the following structure:
...
+* `chain` - (Optional) filter assets by CAIP-10 chain id.
+
+#### Success response body:
+
+* `positions` - list of address positions with the following attributes:
+    * `name` - asset name. e.g. `Ethereum`
+    * `symbol` - native asset or ERC-20 symbol. e.g. `ETH`
+    * `chainId` - CAIP-10 chain id. e.g. `eip155:1`
+    * `value` - asset value in currency calculated from the request argument.
+    * `price` - asset price in currency calculated from the request argument.
```
Comment (arein) [inline docs/specs/servers/blockchain/blockchain-server-api.md]:
```suggestion
    * `price` - price of a single unit of the asset in currency calculated from the request argument.
```

File: docs/specs/servers/blockchain/blockchain-server-api.md [hunk 2]
```diff
@@ -276,3 +276,37 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## Account 
+
+### Balance
+
+Used to lookup address asset balances
```
Comment (arein) [inline docs/specs/servers/blockchain/blockchain-server-api.md]: can you cover whether this includes non-fungible assets?

**Review by arein — APPROVED** [review]

File: docs/specs/servers/blockchain/blockchain-server-api.md [hunk 3]
```diff
@@ -276,3 +276,37 @@ The POST request body should be in JSON format with the following structure:
...
+* `address` - is the address for lookup. eg. `0xab16a96D359eC26a11e2C2b3d8f8B8942d5Bfcdb`
+
+#### Query parameters
+
+* `currency` - currency to calculate the assets value. e.g. `USD`
+* `chain` - (Optional) filter assets by CAIP-10 chain id.
+
+#### Success response body:
+
+* `positions` - list of address positions with the following attributes:
```
Comment (chris13524) [inline docs/specs/servers/blockchain/blockchain-server-api.md]: Assuming we are just forwarding to another API, but this API would return _all_ positions which could be unnecessary and large, and would also not support a custom token ID/address

**Review by chris13524 — APPROVED** [review]
Some comments on the PR that would be great to clarify in the spec, but otherwise looks like good approach so approving

File: docs/specs/servers/blockchain/blockchain-server-api.md (hunk 3 above)
Comment (bkrem) [reply docs/specs/servers/blockchain/blockchain-server-api.md]: Seems a bit disjointed from the naming of the endpoint, wouldn't `balances` fit better? My first instinct reading `positions` is to think trading.

**Review by bkrem — APPROVED** [review]
🚀

File: docs/specs/servers/blockchain/blockchain-server-api.md (hunk 3 above)
Comment (geekbrother) [reply docs/specs/servers/blockchain/blockchain-server-api.md]: Makes sense to me to change it to `balances`. It's changed. Thanks for the suggestion!

File: docs/specs/servers/blockchain/blockchain-server-api.md (hunk 2 above)
Comment (geekbrother) [reply docs/specs/servers/blockchain/blockchain-server-api.md]: Resolved in https://github.com/WalletConnect/walletconnect-specs/pull/206/commits/393a30b86b9852bea474372754bcae4482fa1e4b

**Pull request merged**